app/ante/testutils/testutils.go

```diff
@@ -0,0 +1,155 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+package testutils
+
+import (
+	"math"
+
+	"github.com/stretchr/testify/suite"
+
+	sdkmath "cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	consensustypes "github.com/cosmos/cosmos-sdk/x/consensus/types"
+
+	"github.com/evmos/evmos/v16/app"
+	ante "github.com/evmos/evmos/v16/app/ante"
+	evmante "github.com/evmos/evmos/v16/app/ante/evm"
+	"github.com/evmos/evmos/v16/encoding"
+	"github.com/evmos/evmos/v16/ethereum/eip712"
+	"github.com/evmos/evmos/v16/testutil/integration/evmos/factory"
+	"github.com/evmos/evmos/v16/testutil/integration/evmos/grpc"
+	"github.com/evmos/evmos/v16/testutil/integration/evmos/keyring"
+	"github.com/evmos/evmos/v16/testutil/integration/evmos/network"
+	"github.com/evmos/evmos/v16/types"
+	evmtypes "github.com/evmos/evmos/v16/x/evm/types"
+	feemarkettypes "github.com/evmos/evmos/v16/x/feemarket/types"
+)
+
+type AnteTestSuite struct {
+	suite.Suite
+
+	network   *network.UnitTestNetwork
+	handler   grpc.Handler
+	keyring   keyring.Keyring
+	factory   factory.TxFactory
+	clientCtx client.Context
+
+	anteHandler     sdk.AnteHandler
+	enableFeemarket bool
+	baseFee         *sdkmath.Int
+	enableLondonHF  bool
+	evmParamsOption func(*evmtypes.Params)
+}
+
+const TestGasLimit uint64 = 100000
```

Comment:
This seems to be unused

Reply:
it is used on some Cosmos and EVM anteHandler tests (util_test.go & fee_market_test.go files)